src/helm/benchmark/static/schema_instruction_following.yaml

```diff
@@ -0,0 +1,210 @@
+    lower_is_better: false
+  - name: Understandability
+    display_name: Understandability
+    short_display_name: Understandability
+    description: Whether the answer is easy to comprehend.
+    lower_is_better: false
+  - name: Completeness
+    display_name: Completeness
+    short_display_name: Completeness
+    description: Whether the response provides a comprehensive answer to the question or request.
+    lower_is_better: false
+  - name: Conciseness
+    display_name: Conciseness
+    short_display_name: Conciseness
+    description: Whether the response is concise and to the point, without unnecessary repetition or elaboration.
+    lower_is_better: false
+  - name: Harmlessness
+    display_name: Harmlessness
+    short_display_name: Harmlessness
+    description: Whether the response avoids using toxic language or helping the user with harmful goals.
+    lower_is_better: false
+
+############################################################
+metric_groups:
+  - name: instruction_following_metrics
+    display_name: Instruction Following
+    metrics:
+      - name: Helpfulness
+        split: ${main_split}
+      - name: Understandability
+        split: ${main_split}
+      - name: Completeness
+        split: ${main_split}
+      - name: Conciseness
+        split: ${main_split}
+      - name: Harmlessness
+        split: ${main_split}
+
+############################################################
+run_groups:
+  - name: instruction_following
+    display_name: Instruction Following
+    description: Given an open-ended instruction in natural language, the goal is to produce a text response that is helpful, understandable, complete, concise and harmless.
+    category: Evaluations
+    subgroups:
+      - anthropic_hh_rlhf
+      - grammar
+      - koala
+      - open_assistant
+      - self_instruct
+      - vicuna
+
+  - name: anthropic_hh_rlhf
+    display_name: Anthropic RLHF dataset
+    short_display_name: Anthropic RLHF dataset
+    description: The dialogue datasets released by Anthropic to facilitate research in model helpfulness and harmlessness ([Bai et al., 2022](https://arxiv.org/pdf/2204.05862.pdf); [Ganguli et al., 2022](https://arxiv.org/pdf/2209.07858.pdf)). We only use the first utterance of each dialogue.
+    metric_groups:
+      - instruction_following_metrics
+    environment:
+      main_name: Helpfulness
+      main_split: test
+    taxonomy:
+      task: open-ended instruction following
+      what: "Human-LM dialogues and preference labels"
+      who: "Workers from MTurk and Upwork, language models from Anthropic"
+      when: "2022"
+      language: English
+
+  # Ideally, the name should be "best_chatgpt_prompts".
+  # But unfortunately the group name in the results is "grammar",
+  # so the schema has to match the same group name.
+  # TODO: Change the group name in the "grammar" run spec, and then change this group name.
```

Comment:
Would changing this require regenerating the benchmark output files?  If so, do we want to make this change before deploying?

Reply:
The changes are non-trivial so I would probably have to spend a day if we want to change "grammar" to "best_chatgpt_prompts". We can't re-run helm-run easily, so we would have to edit the JSON files post-hoc.

In any case, I would prefer to do it in a different pull request.